### ISSUE TITLE
Missing aria-label attributes to buttons Play Stop Pause Resume

### ISSUE THREAD

**Issue by hlebegue** [AndrewKeig/react-speech#55]
Could you please add the aria-label attributes to the buttons to allow screen readers to read the functionality of the buttons

This is somewhat urgent, else please advise if this component will get any updates 
Thank you
